### ISSUE TITLE
Small debugging help

### ISSUE THREAD

**Issue by matentzn** [gouttegd/grainyhead#21]
After a year of use, I encountered a new strange error:

```
grainyhead -s oba metrics --from=2023-06-01 --to=2024-05-31 --team monarch-obo-squad > stats/grainyhead_oba.md
Traceback (most recent call last):
  File "/Users/matentzn/ws/rppr_stats/.venv/bin/grainyhead", line 8, in <module>
    sys.exit(grh())
             ^^^^^
  File "/Users/matentzn/ws/rppr_stats/.venv/lib/python3.11/site-packages/click/core.py", line 1157, in __call__
    return self.main(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/matentzn/ws/rppr_stats/.venv/lib/python3.11/site-packages/click/core.py", line 1078, in main
    rv = self.invoke(ctx)
         ^^^^^^^^^^^^^^^^
  File "/Users/matentzn/ws/rppr_stats/.venv/lib/python3.11/site-packages/click_shell/core.py", line 164, in invoke
    ret = super(Shell, self).invoke(ctx)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/matentzn/ws/rppr_stats/.venv/lib/python3.11/site-packages/click/core.py", line 1688, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
                           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/matentzn/ws/rppr_stats/.venv/lib/python3.11/site-packages/click/core.py", line 1434, in invoke
    return ctx.invoke(self.callback, **ctx.params)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/matentzn/ws/rppr_stats/.venv/lib/python3.11/site-packages/click/core.py", line 783, in invoke
    return __callback(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/matentzn/ws/rppr_stats/.venv/lib/python3.11/site-packages/click/decorators.py", line 45, in new_func
    return f(get_current_context().obj, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/matentzn/ws/rppr_stats/.venv/lib/python3.11/site-packages/incenp/grainyhead/main.py", line 360, in metrics
    metrics = reporter.get_report(selector, start, end, period)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/matentzn/ws/rppr_stats/.venv/lib/python3.11/site-packages/incenp/grainyhead/metrics.py", line 51, in get_report
    return self._get_report_for_period(selectors, start, end)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/matentzn/ws/rppr_stats/.venv/lib/python3.11/site-packages/incenp/grainyhead/metrics.py", line 75, in _get_report_for_period
    report = self.get_single_report(item_filter)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/matentzn/ws/rppr_stats/.venv/lib/python3.11/site-packages/incenp/grainyhead/metrics.py", line 121, in get_single_report
    contributors.extend([e.actor.login for e in issues_closes])
                        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/matentzn/ws/rppr_stats/.venv/lib/python3.11/site-packages/incenp/grainyhead/metrics.py", line 121, in <listcomp>
    contributors.extend([e.actor.login for e in issues_closes])
                         ^^^^^^^^^^^^^
AttributeError: 'NoneType' object has no attribute 'login'
make[1]: *** [stats/grainyhead_oba.md] Error 1
make: *** [p1-rppr-2024] Error 2
```

Note that 

```
grainyhead -s mondo metrics --from=2023-06-01 --to=2024-05-31 --team monarch-ontology-team > stats/grainyhead_mondo.md
```

works fine

- the token set in the config is the same for both
- `monarch-obo-squad` exists

Any idea on a quick fix @gouttegd? I will have to fix it for sure (report deadline tomorrow) but thought i would ask first :D

**Comment by gouttegd** [comment]
Pretty sure I’ve seen that before. I believe this is what I fixed in commit a01870aab4f29109ca77cbef2ae08c02c5b5108b, but I have made no release since then.

Can you try using GrainyHead from the tip of the master branch (e.g. something like `python -m pip install -U git+https://github.com/gouttegd/grainyhead.git`, possibly in an isolated environment if you don’t want to mess with your normal install)? If you confirm it works, I can make a bug fix 0.3.2 release.

**Comment by matentzn** [comment]
That was it. thanks!
